app/views.py

```diff
@@ -1309,4 +1307,6 @@ def all_folders(request, company_id):
             cache.set(cache_key, folders_list, timeout=60)
         except:
             return Response(status.HTTP_500_INTERNAL_SERVER_ERROR)
+    if request.method == "PUT":
```

Comment:
What is this method and code block doing here? Remove it

Reply:
Okay